### ISSUE TITLE
MySQL startup issue breaks Jenkins run

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/kg-idg#94]
### Describe the bug

The database startup portion of the start of the Jenkins pipeline encounters this error:

```
21:44:43  + sudo /etc/init.d/postgresql start
21:44:43   * Starting PostgreSQL 12 database server
21:44:47     ...done.
21:44:47  + break
[Pipeline] sh
21:44:48  + sudo /etc/init.d/mysql start
21:44:48   * Starting MySQL database server mysqld
21:44:48  su: warning: cannot change directory to /nonexistent: No such file or directory
21:45:20     ...fail!
21:45:20  + sleep 60
[Pipeline] sh
21:46:28  + sudo /etc/init.d/postgresql status
21:46:28  12/main (port 5432): online
[Pipeline] echo
21:46:30  PostgreSQL server status:
[Pipeline] sh
21:46:30  + pg_isready -h localhost -p 5432
21:46:30  localhost:5432 - accepting connections
[Pipeline] sh
21:46:31  + sudo /etc/init.d/mysql status
21:46:31   * MySQL is stopped.
```

### To Reproduce

Started after merging commits 557cf81f5eb85a3667af5d0c428e25180c4ba972 and 0390d4274be3da489794831f99c8c3d8870fec5b

These introduce `ensmallen` as a new req, but that's Python-specific, and this happens outside the Python context.

Could the Docker image have changed? ([No, it is unchanged](https://hub.docker.com/layers/178711216/caufieldjh/ubuntu20-python-3-8-5-dev/4-with-dbs-v6/images/sha256-4cfca4300e65a0c7edb8844b7798161fcf6f7603d62c38aaea5975330f44ae43?context=repo))

## Expected behavior

In the previous run, starting the DB servers looks like this:
```
[2022-05-12T15:15:12.946Z] + sudo /etc/init.d/postgresql start
[2022-05-12T15:15:12.946Z]  * Starting PostgreSQL 12 database server
[2022-05-12T15:15:16.175Z]    ...done.
[2022-05-12T15:15:16.175Z] + break
[Pipeline] sh
[2022-05-12T15:15:16.574Z] + sudo /etc/init.d/mysql start
[2022-05-12T15:15:16.574Z]  * Starting MySQL database server mysqld
[2022-05-12T15:15:16.574Z] su: warning: cannot change directory to /nonexistent: No such file or directory
[2022-05-12T15:15:21.768Z]    ...done.
[2022-05-12T15:15:21.768Z] + break
[Pipeline] sh
[2022-05-12T15:15:22.167Z] + sudo /etc/init.d/postgresql status
[2022-05-12T15:15:22.167Z] 12/main (port 5432): online
[Pipeline] echo
[2022-05-12T15:15:22.350Z] PostgreSQL server status:
[Pipeline] sh
[2022-05-12T15:15:22.704Z] + pg_isready -h localhost -p 5432
[2022-05-12T15:15:22.704Z] localhost:5432 - accepting connections
[Pipeline] sh
[2022-05-12T15:15:23.108Z] + sudo /etc/init.d/mysql status
[2022-05-12T15:15:23.108Z]  * /usr/bin/mysqladmin  Ver 8.0.27-0ubuntu0.20.04.1 for Linux on x86_64 ((Ubuntu))
[2022-05-12T15:15:23.108Z] Copyright (c) 2000, 2021, Oracle and/or its affiliates.
[2022-05-12T15:15:23.108Z] 
[2022-05-12T15:15:23.108Z] Oracle is a registered trademark of Oracle Corporation and/or its
[2022-05-12T15:15:23.108Z] affiliates. Other names may be trademarks of their respective
[2022-05-12T15:15:23.108Z] owners.
[2022-05-12T15:15:23.108Z] 
[2022-05-12T15:15:23.108Z] Server version		8.0.27-0ubuntu0.20.04.1
[2022-05-12T15:15:23.108Z] Protocol version	10
[2022-05-12T15:15:23.108Z] Connection		Localhost via UNIX socket
[2022-05-12T15:15:23.108Z] UNIX socket		/var/run/mysqld/mysqld.sock
[2022-05-12T15:15:23.108Z] Uptime:			7 sec
[2022-05-12T15:15:23.108Z] 
[2022-05-12T15:15:23.108Z] Threads: 2  Questions: 8  Slow queries: 0  Opens: 117  Flush tables: 3  Open tables: 36  Queries per second avg: 1.142
```

### Version

e0ae2b21c41895ee091275b68cfbcc7ea1d199ad

**Comment by caufieldjh** [comment]
This has happened before, see #63

**Comment by caufieldjh** [comment]
A build started on May 23 did not appear to have this problem - mysql started without issue. Strange!

**Comment by caufieldjh** [comment]
This happened on the Oct 1 build.

**Comment by caufieldjh** [comment]
Should be fixed by #110
